src/android/CameraPreviewFragment.java

```diff
@@ -114,7 +119,26 @@ public void startCamera() {
         }
     }
 
+    public void torchSwitch(boolean state, TorchCallback torchCallback) {
```

Comment:
Needing your expertise to find the proper variable name to replace state. state is okay but the way we formulated the torchCallback line 31, I find `state` as a boolean value inappropriate in the ternary. To note that `state` replaced `on`. We got `switchedOn`..any other names @azharbeebeejaun  @zafirskthelifehacker @dinitri-ragoo ?

Reply:
isOn ? isLit ? 🔥

Reply:
torchOn pour être plus verbeux s'il le faut.